code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -191,13 +191,13 @@ emp_act
 		return 0
 
 	if(istype(I.attack_verb, /list) && I.attack_verb.len)
-		visible_message("<span class='danger'>[src] has been [pick(I.attack_verb)] in the [hit_area] with [I.name] by [user]!</span>", \
-			"<span class='userdanger'>You have been [pick(I.attack_verb)] in the [hit_area] with [I.name] by [user]!</span>")
+		visible_message("<span class='danger'>[src] has been [pick(I.attack_verb)] in the [hit_area] with \the [I.name] by [user]!</span>", \
+			"<span class='userdanger'>You have been [pick(I.attack_verb)] in the [hit_area] with \the [I.name] by [user]!</span>")
 	else
-		visible_message("<span class='danger'>[src] has been attacked in the [hit_area] with [I.name] by [user]!</span>", \
-			"<span class='userdanger'>You have been attacked in the [hit_area] with [I.name] by [user]!</span>")
+		visible_message("<span class='danger'>[src] has been attacked in the [hit_area] with \the [I.name] by [user]!</span>", \
+			"<span class='userdanger'>You have been attacked in the [hit_area] with \the [I.name] by [user]!</span>")
```

Comment:
`/the` is only concerned with the next word in the string, and whether it shows up is just based on whether the first letter of the word is a capital letter or not. It'd probably even work without any insertions at all, a la `...has been attacked with /the Test by...`
Plus I tested it like that and it works.